tools/dev/setup-bochs.sh

```diff
@@ -28,16 +28,21 @@ export WORKDIR=$CURDIR/nanvix-toolchain
 mkdir -p $WORKDIR
 cd $WORKDIR
 
+
+export CFLAGS=-pipe
+export CXXFLAGS=-pipe
+
 # Get required packages.
-apt-get install libgtk2.0-dev
+apt-get install -y libncurses5-dev
 
 # Get bochs.
 wget "http://sourceforge.net/projects/bochs/files/bochs/2.6.8/bochs-2.6.8.tar.gz"
 
 # Build Bochs
 tar -xvf bochs-2.6.8.tar.gz
 cd bochs-2.6.8/
-./configure --enable-x86-debugger --enable-debugger --enable-debugger-gui
+patch < $CURDIR/tools/dev/gdbstub.patch
```

Comment:
Hey!

This patch fixes a behavior in Bochs's GDB Stub that causes the signal zero to be emitted to GDB each time a page fault occurs with signal zero (which GDB cannot ignore).

Basically, this patch adds a condition to emit SIGSEGV (instead of signal zero) when Bochs stops because of a page fault (and other reasons not really clear, but definitely not related to step by step and/or  breakpoints). As SIGSEGV can be configured to be ignored by GDB,  with `handle SIGSEGV nostop nopass noprint` in GDB, we can finally ignore/block these stops caused by the page fault. =)